packages/tangle/timemanager.go

```diff
@@ -53,8 +56,9 @@ func NewTimeManager(tangle *Tangle) *TimeManager {
 
 	// initialize with Genesis
 	t.lastConfirmedMessage = LastConfirmedMessage{
-		MessageID: EmptyMessageID,
-		Time:      time.Unix(DefaultGenesisTime, 0),
+		MessageID:     EmptyMessageID,
+		Time:          time.Unix(DefaultGenesisTime, 0),
```

Comment:
Let's add some comments on these fields, as it may be confusing now with two times.

Reply:
Done